capstone/src/main/webapp/form-loader.js

```diff
@@ -0,0 +1,38 @@
+function callScriptFunction() {
+    var scriptId = "1Z_td2xr1Hq9loDzSdojcCS_3qFwKJR3apBuR2zmcyUVpdhqvfJWyMMYZ";
+  
+    // Call the Apps Script API run method
+    //   'scriptId' is the URL parameter that states what script to run
+    //   'resource' describes the run request body (with the function name
+    //              to execute)
+    gapi.client.script.scripts.run({
+    'scriptId': scriptId,
+    'resource': {
+        "function": "createForm",
+        "parameters": [],
+        "devMode": true
+    }
+    }).then(function(resp) {
+        var result = resp.result;
+        if (result.error && result.error.status) {
+            // The API encountered a problem before the script started executing
+            appendPre('Error calling API:');
```

Comment:
As above, appending text nodes is not desirable error reporting to the user. In the meantime, for something like this, just console.error it

Reply:
Got it.